src/supertux/error_handler.cpp

```diff
@@ -60,7 +71,58 @@ ErrorHandler::handle_error(int sig)
 void
 ErrorHandler::print_stack_trace()
 {
-#ifdef __GLIBC__
+#ifdef WIN32
+  std::string stacktrace = "";
+  std::string msg = "SuperTux has encountered an unrecoverable error!\n";
+
+  {
+    std::stringstream stacktrace_stream;
+
+    // Initialize symbols
+    SymSetOptions(SYMOPT_UNDNAME | SYMOPT_DEFERRED_LOADS);
+    if (!SymInitialize(GetCurrentProcess(), NULL, TRUE))
+    {
+      goto stacktrace_error;
```

Comment:
Instead of using `goto`, perhaps getting the stacktrace from symbols could be moved to a different function (only compiled for `WIN32`) instead?